operations/workplace/README.md

```diff
@@ -4,31 +4,31 @@ description: Description of norms on Mattermost Community server
 
 # Workplace
 
-## What is the Mattermost pre-release server?
+## What is the Mattermost Community Server?
```

Comment:
This section is actually quite outdated. Today, we have three separate "community" servers:

- [community.mattermost.com](https://community.mattermost.com) running our last stable release
- [community-release.mattermost.com](https://community-release.mattermost.com) running our upcoming stable release)
- [community-daily.mattermost.com](https://community-daily.mattermost.com) running the latest builds of Mattermost's software

Reply:
We could update the text to say

```
## What is the Mattermost Community Server?

The Mattermost community server runs the latest stable build of Mattermost's software, including all open source code for Mattermost Team Edition as well as commercial code for Mattermost Enterprise Edition.

It has two objectives:

* Provide a workplace for Mattermost staff.
* Engage with our [communities ](../../contributors/contributors/community.md), including users, contributors and partners.

**The Mattermost community server is not a production server and has occasional downtime.**

You can reach the server at:

{% embed url="https://community.mattermost.com" caption="" %}

The server is also available via the following URLs, primarily used to test pre-release features:

 - [https://community-release.mattermost.com](https://community-release.mattermost.com), which runs our upcoming stable release. This server includes work-in-progress features, will contain bugs and have occasional downtime. This server receives regular pushes of features and/or bug fixes of the next release for testing purposes
 - [https://community-daily.mattermost.com](https://community-daily.mattermost.com), which runs latest builds of Mattermost's software, including all pre-release features of our core products. It is the most unstable version of the Mattermost community server. Bugs and occasional downtime is expected as this server receives daily pushes of features and/or bug fixes for testing purposes.
```

Reply:
Thanks Jason, I've updated it.